site/src/main/paradox/migrations/v0.12.0-Migration-Guide.md

```diff
@@ -79,3 +79,35 @@ case class MyClass(s: String, i: Int)
 val scoll: SCollection[MyClass] = ???
 scoll.saveAsDynamicParquetAvroFile("gs://output/") { m => s"/${m.s}/${m.i}"}
 ```
+
+## Parquet Reads
+
+Alongside the existing Parquet read implementation ("legacy Parquet"), we're concurrently offering a new Parquet read implementation that uses Beam's new [SplittableDoFn](https://beam.apache.org/blog/splittable-do-fn/)
+API. Legacy Parquet is still the default read format, but can enable the new implementation in your `Configuration`:
```

Comment:
strange to state we default to legacy. would be preferable that we offer new implementation instead :)

Reply:
I know 😢 I'm just concerned about performance degradation if users can't immediately update to RunnerV2. Let's aim to make it the default by 0.13.x?